index.html

```diff
@@ -1298,33 +1281,18 @@
   goals and preferences about use of data about them.</span>
 </div>
 
-Agents should be careful, cautious, and conservative in applying data minimization to any
-[=ancillary use=], but also not burden the user with additional [=privacy
-labor=]. To that end, user agents may employ user research, solicitation of general preferences, and
```

Comment:
Removing reference to the concept of privacy labor from this section seems unhelpful. It's been raised regularly, both internal to our group and from external parties. It also explains the reasoning for why we provide examples of general or global user signals.